assets/sass/okta-sign-in.scss

```diff
@@ -32,10 +47,33 @@
   @import 'modules/footer';
   @import 'modules/enroll';
   @import 'modules/app-login-banner';
-  @import 'modules/okta-footer';
   @import 'modules/accessibility';
   @import 'modules/registration';
-  @import 'ie';
+
+  &.can-remove-beacon {
```

Comment:
weren't these better in beacon.scss? Since they are only for the beacon.

Reply:
Unfortunately, the scss lint in okta-core throws an error if the top level selector starts with &, but sign in widget lets it through. This is an error in sass module itself and they suggest updating to 3.0 which I do not want to do at this point.

Reply:
nit: we could move it to beacon file and just add the whole selector, can't we? 
```
#okta-sign-in.can-remove-beacon {
```

Reply:
well, we can't as beacon scss is imported inside #okta-sign-in and doing the above would make it 

```#okta-sign-in #okta-sign-in.can-remove-beacon {```

which would be wrong :(

Reply:
sad... got it. Thanks for the explanation